src/components/HeroSection/HeroSection.css

```diff
@@ -0,0 +1,69 @@
+.hero-wrapper{
+    grid-area: hero-section;
+    min-height: calc(90vh);
+    padding: 1rem;
+    max-width: 100%;
+    gap:0.4rem;
+}
+.hero-image{
+    margin: 0;
+    max-height: 50vh;
+}
+.responsive-img{
+    max-inline-size: 100%;
+    block-size: auto;
+    font-size: inherit;
+    display: block;
+    max-width: 100%;
+}
+.hero-heading{
+    max-width: max-content;
+    text-align: center;
+    color: var(--bg-color);
+    font-family:"Roboto-slab",serif;
+}
+
+.hero-links-wrapper{
+    margin-top: 1.5rem;
+    gap: 1rem;
+    max-width: 100%;
+}
+
+.hero-links-wrapper a{
```

Comment:
i believe you will require this style for many links, so instead make a utility class

Reply:
Sure will do that while refactoring it!!